gvr-remote-scripting/app/src/main/java/org/gearvrf/sample/remote_scripting/GearVRScriptingMain.java

```diff
@@ -15,30 +15,34 @@
 
 package org.gearvrf.sample.remote_scripting;
 
-import org.gearvrf.GVRMain;
+import android.view.Gravity;
+
 import org.gearvrf.GVRContext;
+import org.gearvrf.GVRMain;
 import org.gearvrf.GVRScene;
 import org.gearvrf.scene_objects.GVRTextViewSceneObject;
-import android.view.Gravity;
 import org.gearvrf.script.GVRScriptManager;
 
-public class GearVRScriptingMain extends GVRMain
-{
+import smcl.samsung.com.debugwebserver.DebugWebServer;
 
+public class GearVRScriptingMain extends GVRMain {
+    private static final String TAG = GearVRScriptingMain.class.getSimpleName();
+    private static final int DEBUG_SERVER_PORT = 5000;
+    DebugWebServer server;
     @Override
     public void onInit(GVRContext gvrContext) {
-        gvrContext.startDebugServer();
         GVRScene scene = gvrContext.getNextMainScene();
 
         // get the ip address
         GearVRScripting activity = (GearVRScripting) gvrContext.getActivity();
         String ipAddress = activity.getIpAddress();
-        String telnetString = "telnet " + ipAddress + " 1645";
+        String debugUrl = "http://" + ipAddress + ":" + DEBUG_SERVER_PORT;
```

Comment:
added a telnet string and also started the telnet debug server. Both the telnet and web consoles would work now.